src/view/draggable/connected-draggable.js

```diff
@@ -203,7 +217,9 @@ const mapDispatchToProps: DispatchProps = {
 // that `connect` provides.
 // It avoids needing to do it own within `Draggable`
 export default connect(
-  makeMapStateToProps(),
+  // returning a function to ensure each
+  // Draggable gets its own selector
+  makeMapStateToProps,
```

Comment:
This was previously an error that caused needless memoization cache busts in certain circumstances. Now returning a function that returns selectors to ensure that each component gets its own selector